index.html

```diff
@@ -3,10 +3,10 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="src/style.css">
     <title>Virtual Keyboard</title>
 </head>
 <body>
-    <script src="/src/script.js"></script>
+    <script src="src/script.js"></script>
 </body>
 </html>
```

Comment:
add emty line

Reply:
Install ESLint for html and remove errors